src/parse.rs

```diff
@@ -9,7 +9,12 @@ pub fn u32(tree: &Element) -> Option<u32> {
         // Handle strings in the binary form of:
         // #01101x1
         // along with don't care character x (replaced with 0)
-        u32::from_str_radix(&str::replace(&text["#".len()..], "x", "0"), 2).ok()
```

Comment:
I'd prefer

```rust
&text.to_lowercase()["#".len()..]
```

Reply:
That is a much better approach. I can adjust that.